webpack/assets/javascripts/react_app/components/charts/Chart.js

```diff
@@ -30,7 +30,9 @@ class Chart extends React.Component {
   }
 
   componentWillUnmount() {
-    this.chart = this.chart.destroy();
+    if (this.hasData()) {
```

Comment:
semi related, most likely should be done in refactoring to charts components to be able to be used in storybook... /cc @gailsteiger